x-pack/plugins/ml/public/application/components/model_snapshots/revert_model_snapshot_flyout/revert_model_snapshot_flyout.tsx

```diff
@@ -0,0 +1,396 @@
+                      dropdownDisplay: (
+                        <>
+                          <strong>{s.snapshot_id}</strong>
+                          <EuiText size="s" color="subdued">
+                            <p className="euiTextColor--subdued">{s.description}</p>
+                          </EuiText>
+                        </>
+                      ),
+                    }))
+                    .reverse()}
+                  valueOfSelected={currentSnapshot.snapshot_id}
+                  onChange={onSnapshotChange}
+                  itemLayoutAlign="top"
+                  hasDividers
+                />
+              </EuiFormRow>
+              <EuiHorizontalRule margin="m" />
+              <EuiSpacer size="l" />
+            </>
+          )}
+
+          <EventRateChart
+            eventRateChartData={eventRateData}
+            anomalyData={anomalies}
+            loading={chartReady === false}
+            height={'100px'}
+            width={'100%'}
+            fadeChart={true}
+            overlayRanges={[
+              {
+                start: currentSnapshot.latest_record_time_stamp,
+                end: job.data_counts.latest_record_timestamp,
+                color: '#ff0000',
+              },
+            ]}
+          />
+
+          <EuiSpacer size="l" />
+          <EuiSpacer size="l" />
+
+          <EuiCallOut
+            title={i18n.translate(
+              'xpack.ml.newJob.wizard.revertModelSnapshotFlyout.warningCallout.title',
+              {
+                defaultMessage: 'Anomaly data will be deleted',
+              }
+            )}
+            color="warning"
+            iconType="alert"
+          >
+            <FormattedMessage
+              id="xpack.ml.newJob.wizard.revertModelSnapshotFlyout.warningCallout.contents"
+              defaultMessage="All anomaly detection results after {date} will be deleted."
+              values={{ date: formatDate(currentSnapshot.latest_record_time_stamp, TIME_FORMAT) }}
+            />
+          </EuiCallOut>
+
+          <EuiHorizontalRule margin="xl" />
+
+          <EuiFormRow
+            fullWidth
+            helpText="Reopen job and replay analysis after the revert has been applied."
+          >
+            <EuiSwitch
+              id="replaySwitch"
+              label={i18n.translate(
+                'xpack.ml.newJob.wizard.revertModelSnapshotFlyout.replaySwitchLabel',
+                {
+                  defaultMessage: 'Replay analysis',
+                }
+              )}
+              checked={replay}
+              onChange={(e) => setReplay(e.target.checked)}
+            />
+          </EuiFormRow>
+
+          {replay && (
+            <>
+              <EuiFormRow
+                fullWidth
+                helpText={
+                  runInRealTime
+                    ? 'Job will continue to run until manually stopped. All new data added to the index will be analyzed.'
```

Comment:
This text needs internationalizing.

Reply:
Testing this, I find it confusing the way the help text changes below the switch. I think it would be better if it always described what happens if you enable the switch i.e. `Job will continue to run until manually stopped. All new data added to the index will be analyzed.`